### DISCUSSION TITLE
Any plans on integrating OpenGL somehow to make them all portable?

### DISCUSSION THREAD

**Discussion by odarcan** [jart/cosmopolitan#960] [Ideas]
Any plans on integrating OpenGL somehow to make them all portable?

**Comment by mundusnine** [comment]
> Any plans on integrating OpenGL somehow to make them all portable?

You can already add OpenGL support with cosmo_dlopen. Versioning OpenGL would be a crazy endeavor and imo doesn't make sense to add to cosmopolitan as it's just a libc(a little more of course).
